Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -254,17 +254,20 @@ public void SearchValidationsRecursive(DependencyObject depObj)
 
         private void UpdatePrevNextButton()
         {
-
             if (mWizard.IsLastPage())
             {
                 xNextButton.IsEnabled = false;
                 xFinishButton.IsEnabled = true;
+                if (mWizard.DisableBackBtnOnLastPage == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4375598)